openhtmltopdf-core/src/main/java/com/openhtmltopdf/css/newmatch/CascadedStyle.java

```diff
@@ -227,23 +227,17 @@ public IdentValue getIdent(CSSName cssName) {
      *
      * @return Iterator over a set of properly cascaded PropertyDeclarations.
      */
-    public java.util.Iterator getCascadedPropertyDeclarations() {
-        List list = new ArrayList(cascadedProperties.size());
-        Iterator iter = cascadedProperties.values().iterator();
-        while ( iter.hasNext()) {
-            list.add(iter.next());
-        }
-        return list.iterator();
+    public java.util.Collection<PropertyDeclaration> getCascadedPropertyDeclarations() {
+        return cascadedProperties.values();
```

Comment:
This collection here is always directly consumed and iterated over. There was no sense in copying this...